Ecosystem Partners/Marketplace Guides/getting-started-with-softnas-cloud-file-gateway-partner-template.md

```diff
@@ -65,6 +64,7 @@ This deployment process for Partner Templates currently requires manual interact
 ### Prerequisite
 - Access to the CenturyLink Cloud platform as an authorized user.
 - Identify a Network VLAN you want the SoftNAS instance/s to reside on
+- Do you want to deploy this in a high availability configuration?  Typically, most users _do not_ deploy a HA configuration, but if you want to there are a few things you should be aware of.  For starters, a SoftNAS HA configuration will consist of 3 SoftNAS appliances.  The primary and secondary will reside in separate data centers and the third will reside in the same data center as the primary.  For more information on SoftNAS high availaility architecture details, please contact [SoftNAS Support](mailto:support@softnas.com).

```

Comment:
i'd argue there is another more common use case for HA in even a single site.  as relying on a single VM to always be online for a SAN is not something i'd recommend as normal SAN systems achieve 5> 9's availability.  the across site approach would more be a primary/secondary DR not HA situation.  i'd edit this to talk about HA & DR completely separate.